.vscode/settings.json

```diff
@@ -15,13 +15,15 @@
     "codegen",
     "Dockerized",
     "Downloadability",
+    "fairplay",
```

Comment:
Not sure about the spelling plugin - but correctly the term would be "FairPlay" and further down "PlayReady" and "Widevine".

Reply:
This option is added using the VSCode lightbulb in all lowercase and checks spelling in all cases, so no need to adjust it here.